client/src/components/Navbar.js

```diff
@@ -1,11 +1,16 @@
-import React,{useContext, useEffect} from 'react';
-import {Toolbar, AppBar, Box, Typography, Link, IconButton, Button} from '@material-ui/core';
+import React,{useContext, useEffect, useState} from 'react';
+import {Toolbar, AppBar, Box, Typography, Link, IconButton, Button,Badge, Popper} from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import { Link as RouterLink } from 'react-router-dom';
 import LocalMallIcon from '@material-ui/icons/LocalMall';
 import AuthContext from '../state_management/AuthContext';
 import {fetchShLists} from '../state_management/actionCreators/shoppingListsActs';
 import ShListsContext from '../state_management/ShListsContext';
+import socketIOClient from "socket.io-client";
+import Notifications from "./Notifications";
+
+
+const ENDPOINT = "http://localhost:3001";
```

Comment:
This ENDPOINT will change on deploy in prod. We will need to add this to .env

Reply:
Good catch!